schema-engine/connectors/sql-schema-connector/src/rendering/models.rs

```diff
@@ -87,7 +87,7 @@ fn render_model(model: ModelPair<'_>, sql_family: SqlFamily) -> renderer::Model<
     }
 
     if model.adds_a_description() {
-        let docs = "This model is commented in the database, and requires an additional setup for migrations: Read more: https://pris.ly/d/database-comments";
+        let docs = "This model or some of its fields is commented in the database, and requires an additional setup for migrations: Read more: https://pris.ly/d/database-comments";
```

Comment:
The formulation feels a bit off but I'm no native speaker, so feel free to ignore. Maybe "The table for this model or at least one of its columns has comments in the database...".

Reply:
fixed in [`443a22e` (#3884)](https://github.com/prisma/prisma-engines/pull/3884/commits/443a22e7c69f276e0f847a31861e6193e0f30042)